pkg/api/volume/handler.go

```diff
@@ -89,19 +95,65 @@ func (h *ActionHandler) Do(ctx *harvesterServer.Ctx) (interface{}, error) {
 	}
 }
 
-func (h *ActionHandler) exportVolume(_ context.Context, imageNamespace, imageDisplayName, imageStorageClassName, pvcNamespace, pvcName string) (interface{}, error) {
+func (h *ActionHandler) validateExportVolume(storageClassName, pvcNamespace, pvcName string) string {
```

Comment:
minor nit, why not make the method return an error and return nil when conditions are met

Reply:
I am ok with using error for this function.
The reason to return string because the caller will return the error w string.
So, it is straightforward to return the string directly.